README.md

```diff
@@ -14,33 +14,40 @@ Full deployment documentation is available at [HospitalRun Deployment](https://c
 5. Install and configure [CouchDB 1.6.1](http://couchdb.apache.org/)
     1. Download and install CouchDB 1.6.1 from http://couchdb.apache.org/#download. Please makes sure you use Couch 1.x and not Couch 2.x as Couch 2.x is not fully supported at this time.
     2. Start CouchDB
         1. If you downloaded the installed app, navigate to CouchDB and double-click on the application.
         2. If you installed CouchDB via Homebrew or some other command line tool, launch the tool from the command line
         3. If you're stuck with the installation, check out the instructions published here: http://docs.couchdb.org/en/1.6.1/install/index.html
-    3. Verify that CouchDB is running by successfully navigating to 127.0.0.1:5984/_utils. If that fails, check the installation guide for CouchDB http://docs.couchdb.org/en/1.6.1/install/index.html
+    3. Verify that CouchDB is running by successfully navigating to 127.0.0.1:5984/_utils. If that fails, check the installation guide for CouchDB (http://docs.couchdb.org/en/1.6.1/install/index.html)
     4. Setup CouchDB for HospitalRun:
         1. Download https://github.com/HospitalRun/hospitalrun-frontend/blob/master/script/initcouch.sh
-        2. If you have just installed CouchDB and have no admin user, simply run `initcouch.sh` with no arguements.  A user `hradmin` will be created with password: `test`.
+        2. If you have just installed CouchDB and have no admin user, simply run `initcouch.sh` with no arguments.  A user `hradmin` will be created with password: `test`.
         2. If you already have a CouchDB admin user, please run `initcouch.sh USER PASS`.  `USER` and `PASS` are the CouchDB admin user credentials.
-6. Copy the config-example.js to config.js in the folder you cloned the HospitalRun repo. If you already had a CouchDB admin user that you passed into the couch script (initcouch.sh USER PASS), then you will need to modify the couchAdminUser and couchAdminPassword values in config.js to reflect those credentials.
-7. If you are on Linux distribution that uses Upstart, there is an upstart script in utils/hospitalrun.conf.  By default this script assumes the server is installed at /var/app/server. This script relies on [forever](https://github.com/foreverjs/forever) which you will need to install via npm: ```npm install -g forever```
+6. Copy the config-example.js to config.js in the folder you cloned the HospitalRun repo. If you already had a CouchDB admin user that you passed into the couch script (initcouch.sh USER PASS), then you will need to modify the `couchAdminUser` and `couchAdminPassword` values in config.js to reflect those credentials.
+7. If you are on Linux distribution that uses Upstart, there is an upstart script in `utils/hospitalrun.conf`.  By default this script assumes the server is installed at `/var/app/server`. This script relies on [forever](https://github.com/foreverjs/forever) which you will need to install via npm: 
+
```

Comment:
This formatting change breaks the indentation of point 7.  This and the following line "* alternatively you can run server using npm's scripts `npm start` (this is not recommended for production usage)." need to be nested under point 7.